.github/workflows/build.yml

```diff
@@ -0,0 +1,70 @@
+name: ci
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+env:
+  IMAGE_NAME: fizzka/docker-phalcon
```

Comment:
Image name: mileschou/phalcon / fizzka/docker-phalcon

I'm not sure what which one to choose.

I guess that my secret access token just can update to my dockerhub repo.

Reply:
"so I'll change paths and labels to your repo." - please look before, after I commit fixes to your repos =)

Reply:
secrets:
![image](https://github.com/MilesChou/docker-phalcon/assets/2384736/e23fab3f-4a7f-44ac-b8f8-0a2d03f833d8)

Reply:
UP

Reply:
"generate new token for github actions and put it"

Ready to go.

Reply:
Did you create token, so I will change image name